tests/beaker_tests/cisco_interface_service/test_interface_service.rb

```diff
@@ -0,0 +1,181 @@
+###############################################################################
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+# TestCase Name:
+# -------------
+# test_interface_service.rb
+#
+# TestCase Prerequisites:
+# -----------------------
+# This is a Puppet Interface resource testcase of vlan_mapping properties,
+# for use with Puppet Agent on Nexus devices.
+# The test case assumes the following prerequisites are already satisfied:
+#   - Host configuration file contains agent and master information.
+#   - SSH is enabled on the N9K Agent.
+#   - Puppet master/server is started.
+#   - Puppet agent certificate has been signed on the Puppet master/server.
+#
+###############################################################################
+#
+#        ****************************************
+#        ** IMPORTANT ADDITIONAL PREREQUISITES **
+#        ****************************************
+#
+# The encapsulation profile vni property has additional prerequisites.
+#
+#  - VDC support
+#  - F3 linecard
```

Comment:
What platform is the F3 linecard for? Might be nice to call out the chassis required.